recipe/meta.yaml

```diff
@@ -31,6 +31,7 @@ requirements:
     - libarchive
     - libiconv
     - nlohmann_json
+    - cpp-filesystem
```

Comment:
You need to ship the license of `cpp-filesystem` in downstream packages since its license is MIT.

Reply:
Ping on this again

Reply:
@conda-forge/core, there's a license violation here.

Reply:
Agreed. We don't have a procedure that I know of for this, though we have discussed this a lot. Given that @wolfv is one core, I would expect him to take care of this quickly. What is your proposed remedy @isuruf?

Reply:
yeah I do have this in the back of my head :)
I don't think we include a "copy or substantial portions of the Software" since cpp-filesystem is a header only library that disappears into binary code.

But anyways, happy to add this.

Reply:
I tried to google this and I could not find a good advisory in the case of compiled MIT code (header only). Do you have a link or reference to what constitutes a substantial portion @isuruf ?

Also I think it would be super awesome if conda-build would somehow be able to do this automatically :)

Reply:
>  Do you have a link or reference to what constitutes a substantial portion @isuruf ?

No, I don't. IANAL. Maybe ask upstream?